CommMicro.py

```diff
@@ -65,86 +65,113 @@ def getPath(fileName):
         self.xfDisp = Display(ui_filename=getPath("MicPlot.ui"))
 
         # Show message on gui
-        self.ui.AcqProg.setText("Select 1 CM and 1 cavity at a time for commissioning. \nLimit plotted waveforms to 30 sec.")         
+        self.ui.AcqProg.setText(
+            "Select 1 CM and 1 cavity at a time for commissioning. \nLimit plotted waveforms to 30 sec.")
 
         # create plot canvases and link to GUI elements
-        TopPlot = MplCanvas(self, width = 20, height =40, dpi=100)
-        BotPlot = MplCanvas(self, width = 20, height =40, dpi=100)
+        TopPlot = MplCanvas(self, width=20, height=40, dpi=100)

```

Comment:
Minor quibble, but variable names should start with a lowercase letter

Reply:
fixed